tests/Unit/ConfigMgrCBDsc.ReverseDsc.tests.ps1

```diff
@@ -2641,6 +2769,96 @@ InModuleScope $script:subModuleName {
                 PSComputerName                            = 'localhost'
             }
 
+            $getGroupDiscoveryEnabled = @{
+                SiteCode = 'Lab'
+                Props    = @(
+                    @{
+                        PropertyName = 'Settings'
+                        Value1       = 'ACTIVE'
+                    }
+                )
+            }
+
+            $getGroupDiscoveryDisabled = @{
+                SiteCode = 'Lab'
+                Props    = @(
+                    @{
+                        PropertyName = 'Settings'
+                        Value1       = 'INACTIVE'
+                    }
+                )
+            }
+
+            $groupDiscovery = @{
+                SiteCode = 'Lab'
+                Include  = 'GroupDiscovery'
+            }
+
+            $invokeGroupDiscoveryEnabled = @{
+                ConfigurationName                   = $null
+                DependsOn                           = $null
+                ModuleName                          = 'ConfigMgrCBDsc'
+                ModuleVersion                       = 1.0.1
+                PsDscRunAsCredential                = $null
+                ResourceId                          = $null
+                SourceInfo                          = $null
+                DayOfMonth                          = $null
+                DayOfWeek                           = $null
+                DeltaDiscoveryMins                  = 50
+                DiscoverDistributionGroupMembership = $true
+                Enabled                             = $true
+                EnableDeltaDiscovery                = $true
+                EnableFilteringExpiredLogon         = $true
+                EnableFilteringExpiredPassword      = $true
+                GroupDiscoveryScope                 = @(
+                    @{
+                        Name           = 'Test1'
+                        LdapLocation   = 'OU=Test1,DC=contoso,DC=com'
+                        Recurse        = $true
+                        PSComputerName = 'localhost'
+                    }
+                )
+                GroupDiscoveryScopeToExclude        = $null
```

Comment:
spacing

Reply:
This lines up on char 53 as above with the other items in the hash table